.gitignore

```diff
@@ -7,3 +7,4 @@ krill.log
 target/
 tmp
 work
+examples/openid_connect_mock.rs
```

Comment:
Did you mean to push this change?

Reply:
I did, but then clippy barks all over the place, although it compiles just fine. It was just a link to the mock auth provider, so that I could `cargo run --examples openid_connect_mock`